src/components/z-menu/index.tsx

```diff
@@ -141,32 +215,60 @@ export class ZMenu {
     const items = this.hostElement.querySelectorAll<HTMLElement>("[slot=item]");
     items.forEach((item) => {
       item.setAttribute("role", "menuitem");
+      item.setAttribute("tabindex", "-1");
       item.dataset.text = item.textContent;
     });
   }
 
+  private focusFirstItemOnKeyUp(): void {
+    const firstElement = this.hostElement.querySelectorAll("[slot='item']")[0] as HTMLElement;
+    if (firstElement) {
+      firstElement.focus();
+      this.currentIndex = 0;
+    }
+  }
+
   private renderMenuLabel(): HTMLButtonElement | HTMLDivElement {
     if (this.hasContent) {
       return (
-        <button
-          class="menu-label"
-          aria-expanded={this.open ? "true" : "false"}
-          aria-label={this.open ? "Chiudi menù" : "Apri menù"}
-          onClick={this.toggle}
-        >
-          <div class="menu-label-content">
-            <slot onSlotchange={this.onLabelSlotChange}></slot>
-            <z-icon name={this.open ? "chevron-up" : "chevron-down"} />
-          </div>
-        </button>
+        <div class="menu-wrapper">
+          <button
+            class="menu-label"
+            aria-expanded={this.open ? "true" : "false"}
```

Comment:
qui potrebbe bastare `this.open`, giusto?

Reply:
no perché` aria-expadend` è un attribute che vuole specificatamente una stringa altrimenti non viene visto nell'albero dell'accessibilità.